app/models/hackbot/conversations/check_in.rb

```diff
@@ -143,6 +151,10 @@ def wait_for_notes(event)
 
       private
 
+      def record_notes(event)
+        data['notes'] = event[:text] unless event[:text] =~ /^(no|nope|nah)$/i
```

Comment:
Problem with the approach you suggested is that it's still not clear that you reject the save if they responded with no with the name `save_notes`. I think that check is a separate "block" of logic that should be explicit and not hidden in a helper.